### PR TITLE
STN-143 : News Node Detail Mobile Image Height adjustment

### PR DESCRIPTION
**Pull request by heatherdesigns**
# [STN-143](https://sparkbox.atlassian.net/browse/STN-143) READY FOR REVIEW 🎉

## Summary
This PR adjusts the mobile styles of the news node image to 1.5x more than its previous setting of 250px.

_Note_ I paired with @meranicosme to get her feedback. We explored the idea of removing the max-height at mobile so that the image would have a height of 100%. She is happy with the new max-height setting of the image at mobile sizes. She also noted that they have been engaged in conversations regarding the cropping of images at different screen sizes.  

## Need Review By (Date)
1/31/2020

## Urgency
low

## Steps to Test
1. To review the work in this PR, you will need to have a News Item Node Detail page. If you already have one, you may skip the steps below.
2. In the Drupal admin panel, go to `Content`
    * Select `Add content`
    * Select `News Item`
    * Fill out all the fields
        * Title
        * Publish date
        * Link to external article
        * News Image
        * Body
        * Categories
    * Save
        * After saving you will be redirected to a page with your newly created content 🎉
3. If you are not seeing all of the items above, update the layout by clicking `Manage Display`
    * Towards the center of the page, click 'Manage Layout`
    * Add the missing blocks and `Save Layout`
    * Click on the `Back to site` button on the top left of the page to view the updated content
    * layout
4. There are concerns on how image styles will works with a square image. This will be of particular concern with event content types. I recommend editing the `News Image` to use an image that either has the same or very similar width and height values.

## Feedback / Opinion requested
At the last demo, `focal point` was presented as a possible solution by the H&S team. To test `focal point` you will need to `edit` the image. I was unable to ascertain any changes with the use of `focal point` and wonder if this might be an issue. Perhaps the use of `object-fit` to contain the image is blocking `focal-point`? Of course, it could be user error on my part.  🤷‍♀ If you are curious, this [video](https://vimeo.com/204099629) explains `focal point`. If `focal-point` is an important consideration, should we address it here or create another card and place it in the backlog for prioritization.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)

### CODE REVIEW

File: docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.three_column_w_image.scss [hunk 1]
```diff
@@ -101,7 +101,11 @@ $hb-sidebar-width: 16.5rem;
       @supports (object-fit: cover) {
         object-fit: cover;
         width: 100%;
-        max-height: 250px;
+        max-height: 23.4375rem;
```
Comment (silverli) [inline docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.three_column_w_image.scss:104]: hmm yeah I still wonder if we shouldn't just let this be 100% height to allow regular image proportions and not crop people's heads off.

**Review by silverli — APPROVED** [review]
Code looks ok, but we might want to run by H&S for final confirmation of this and the focal point question. Screenshots of what I saw:

Screen Shot 2020-02-03 at 10 32 23 AM
Screen Shot 2020-02-03 at 10 32 32 AM

**Comment by heatherdesigns** [comment]
Different picture but we still have some heading cropping / chopping 😆 happening.

Medium screens
medium

Small screens
mobile

File: docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.three_column_w_image.scss (hunk 1 above)
Comment (kaseybon) [reply docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.three_column_w_image.scss:104]: The concern with having a max-height of 100% is that they might have a tall enough image that pushes all text off the screen when scrolling. What if we try a max-height of `90vh` to ensure that there is always some text or part of the page visible?

**Pull request merged**